src/tree-sitter/csharpQueries.ts

```diff
@@ -0,0 +1,52 @@
+import { TreeSitterQueries } from './types.js'
+
+export const csharpQueries: TreeSitterQueries = {
+  defineParameterTypeQueries: [
+    `
+(method_declaration
+  (attribute_list
+    [
+      (attribute
+        name: (identifier) @annotation-name
+        (attribute_argument_list
+          (attribute_argument
+            (verbatim_string_literal) @expression
+          )
+        )
+      )
+      (attribute
+        name: (identifier) @annotation-name
+      )
+    ]
+  )
+  name: (identifier) @name
+  parameters: (parameter_list
+    [
+      (parameter
+        type: (predefined_type)
+        name: (identifier)
+      )
+    ]
+  )
+  (#eq? @annotation-name "StepArgumentTransformation")
+)
+`,
+  ],
+  defineStepDefinitionQueries: [
+    `
+(method_declaration
+  (attribute_list
+    (attribute
+      name: (identifier) @annotation-name
+      (attribute_argument_list
+        (attribute_argument
+          (verbatim_string_literal)
```

Comment:
Try adding `@expression` at the end of this line to capture the (regular) expression of the step definition.

Reply:
Done. Thanks